### PR TITLE
Feature/network selection

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
Add network selector to settings.
Use selected network option to configure helium js and wallet client.
Always read pending txns from mainnet.

### CODE REVIEW

File: src/utils/appDataClient.ts [hunk 1]
```diff
@@ -24,7 +25,27 @@ import { fromNow } from './timeUtils'
...
+    (network === 'helium' && client.network === Network.production)
+  )
+}
+
+export const updateClient = (nextNetwork: string) => {
+  const isSame = compareNetwork(nextNetwork)
+  if (!isSame) {
+    const network =
+      nextNetwork === 'helium' ? Network.production : Network.stakejoy
+    client = new Client(network)
```
Comment (tyler-whitman) [inline src/utils/appDataClient.ts:45]: Could this get reinitialized while things are already partly paged and throw off the cursor? Or do we just call this and everything in the app refreshes?

File: src/App.tsx [hunk 2]
```diff
@@ -86,9 +87,14 @@ const App = () => {
 
   useMount(() => {
     dispatch(restoreAppSettings())
+  })
+
+  useEffect(() => {
+    if (!settingsLoaded) return
```
Comment (tyler-whitman) [inline src/App.tsx:93]: Does fetchAccountSettings only get dispatched on the more screen? Should we also call it here to makes sure it loads and doesnt block the app?

**Review by tyler-whitman — APPROVED** [review]

File: src/utils/appDataClient.ts (hunk 1 above)
Comment (matthewcarlreetz) [reply src/utils/appDataClient.ts:45]: I had the same concern. A couple of lines down I re-init the txn fetchers and after thorough testing, it all seems to work. Could you test when you have a chance? The wallet API is updated now.

File: src/App.tsx (hunk 2 above)
Comment (matthewcarlreetz) [reply src/App.tsx:93]: It gets called in App.tsx by the `useAccountSettings` hook as well.

**Pull request merged**